src/routes/api/accommodationsRoute.js

```diff
@@ -250,7 +248,7 @@ router.get('/', permit(['view accommodations']), getAccommodations);
  *                      description: Availability of wifi
  *                      example: false
  */
-router.get('/:id', permit(['view accommodations']), getOneAccommodation);
```

Comment:
from my understanding this should be protected , as per opening accomodations was for the sake of  landing page . so viewing more details should be protected.

Reply:
@salviosage fixed ✅